packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -45,22 +45,7 @@ class _DeepLinkListViewState extends State<DeepLinkListView>
     super.didChangeDependencies();
     initController();
```

Comment:
this should be `if (!initController()) return;` See dart doc on initController method for explanation